### PR TITLE
linters: fix; unnecessary parentness, if/else => ?, imports

### PR DESCRIPTION
**Pull request by Horki**
@cristicbz small linter fixes, everything compiles and works

### CODE REVIEW

File: game/src/lights.rs [hunk 1]
```diff
@@ -19,7 +19,7 @@ impl Lights {
                 // TODO(cristicbz): Remove this restriction.
                 assert!(self.lights.len() < 255);
                 self.lights.push(light_info.clone());
-                (self.lights.len() - 1)
```
Comment (cristicbz) [inline game/src/lights.rs:22]: is the `_usize` suffix necessary? it seems like it should work without it

**Review by cristicbz — APPROVED** [review]
Just one question on one of the changes, thanks for the PR!

File: game/src/lights.rs (hunk 1 above)
Comment (Horki) [reply game/src/lights.rs:22]: @cristicbz it's really not, just my preference if it's problem I can redo it

**Comment by cristicbz** [comment]
Yes please, seems superfluous. Otherwise air enough on the lint errors

File: game/src/lights.rs (hunk 1 above)
Comment (Horki) [reply game/src/lights.rs:22]: @cristicbz hi, I removed usize suffix

**Comment by cristicbz** [comment]
Thanks! Walking on the street right now :) will merge in a few hours when I
get home

**Comment by cristicbz** [comment]
Merged, thanks!

**Pull request merged**